src/core/Microsoft.PowerToys.Settings.UI.Library/Microsoft.PowerToys.Settings.UI.Library.csproj

```diff
@@ -51,6 +51,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\common\interop\interop.vcxproj" />
+    <ProjectReference Include="..\..\common\ManagedCommon\ManagedCommon.csproj" />
```

Comment:
if we're moving in manage common, we need to test out the installer works here as well.